lib/vagrant/util/powershell.rb

```diff
@@ -220,6 +242,19 @@ def self.powerup_command(path, args, opts)
       def self.reset!
         instance_variables.each(&method(:remove_instance_variable))
       end
+
+      # @private
+      # This is a helper method that provides the PowerShell command to resize
+      # the "console" to prevent output wrapping or truncating. An environment
+      # variable guard is provided to disable the behavior in cases where it
+      # may cause unexpected results (VAGRANT_POWERSHELL_RESIZE_DISABLE)
+      def self.resize_console
+        if ENV["VAGRANT_POWERSHELL_RESIZE_DISABLE"]
```

Comment:
I think this setting should be documented so that it's clear it's available for use.